scenarios/sqs_flag_shop/terraform/lambda.tf

```diff
@@ -1,6 +1,6 @@
 resource "aws_lambda_function" "charging_cash_lambda" {
   function_name = "lambda_function"
-  handler       = "lambda_function.lambda_handler"
+  handler       = "charging_cash_lambda/lambda_function.lambda_handler"
```

Comment:
This probably still will cause issues, where it cannot find the module (did not test)

See other comment: https://github.com/RhinoSecurityLabs/cloudgoat/pull/236/files/15817b08d2c838b34fe248beaae85b2822298e82..970eb85026cee372d61fce01237ed448310de63e#r1405442230